pkg/runner/gas.go

```diff
@@ -51,7 +51,7 @@ func gasInitialization(memory *mem.Memory) error {
 		return err
 	}
 
-	preCostTokenTypes := []TokenGasCost{PedersenToken, BitwiseToken, EcOpToken, PoseidonToken, AddModToken, MulModToken}
+	preCostTokenTypes := []TokenGasCost{PedersenToken, PoseidonToken, BitwiseToken, EcOpToken, AddModToken, MulModToken}
```

Comment:
I'm assuming the order here is important since you are writing the costs to the memory, is that correct? Can you leave a comment here for it? Also with the source?

Reply:
yes, done